src/content/reference/react/Profiler.md

```diff
@@ -32,41 +32,41 @@ Wrap a component tree in a `<Profiler>` to measure its rendering performance.
 
 #### Props {/*props*/}
 
-* `id`: A string identifying the part of the UI you are measuring.
-* `onRender`: An [`onRender` callback](#onrender-callback) that React calls it every time components within the profiled tree update. It receives information about what was rendered and how much time it took.
+* `id`: Một chuỗi ký tự để xác định phần giao diện bạn muốn đo lường.
+* `onRender`: Một [callback `onRender`](#onrender-callback) mà React gọi mỗi khi các thành phần bên trong tree cập nhật. Nó nhận thông tin về những gì được render và mất bao nhiêu thời gian để render.
 
-#### Caveats {/*caveats*/}
+#### Những điều cần chú ý {/*caveats*/}
 
-* Profiling adds some additional overhead, so **it is disabled in the production build by default.** To opt into production profiling, you need to enable a [special production build with profiling enabled.](https://fb.me/react-profiling)
+* Việc đo lường sẽ làm giảm hiệu suất, vì vậy **mặc định nó sẽ luôn được vô hiệu hoá trong bản build production.** Để đo lường ở production, bạn cần kích hoạt [tính năng đo lường trong một bản build production đặc biệt.](https://fb.me/react-profiling)
 
 ---
 
 ### `onRender` callback {/*onrender-callback*/}
 
-React will call your `onRender` callback with information about what was rendered.
+React sẽ gọi call `onRender` callback với thông tin về những gì được render.
```

Comment:
Seems we don't need an extra "call" here.

Reply:
Fixed ! Thank you